### ISSUE TITLE
paragraphs getting removed in reading mode

### ISSUE THREAD

**Issue by incantatem1** [isaacfreeman/kakano-obsidian-theme#11]
can you add an option for them to stay? i have some chat history like documents where the lack of paragraphs looks really confusing in this theme, otherwise i'm a quite big fan of it. am i missing an option somewhere which does that? i honestly have no idea and i'm also not sure what CSS element does this, i'd prefer it as a toggleable option

**Comment by isaacfreeman** [comment]
Thanks! This was just a bug in my view: it doesn't need an option it should just keep paragraphs in reading mode.

I'm not sure when this happened, as I seldom use reading mode myself, but try version 3.0.7 and let me know if it's working for you. It should behave similarly to the default theme: paragraph breaks keep their spacing, but multiple breaks following each other collapse into a single break in reading mode.

**Comment by incantatem1** [comment]
just updated, paragraphs are there now!

what do you mean with the last sentence though? i never observed such behaviour of paragraphs....  do you mean like if i press enter multiple times in a row?

**Comment by isaacfreeman** [comment]
>i actually liked how narrow the editor content was in the previous version in reading mode, can you restore that while keeping the paragraphs? i feel like it was part of this themes unique character

It's always interesting which parts appeal to people! Do you like it narrower just in reading more, or in both reading mdoe and live preview?

I've been trying to keep reading mode and live preview as similar as I can make them, but since I don't use reading mode much anyway, and there are limitations preventing them from being perfectly identical, perhaps I should understand more about what people want from reading mode.

> it seems to add additional paragraphs in editing mode now

The difference in behaviour comes if you have two consecutive empty lines. In editing modes (live preview mode or source mode) these will appear as two separate lines, but in reading mode they'll be collapsed into a single empty line. This derives from the underlying behaviour of Markdown: reading mode is equivalent to rendering Markdown as HTML, and collapsing empty whitespace is standard HTML behaviour.

**Comment by incantatem1** [comment]
hmm, i liked the narrowness in reading mode specifically. but i actually wouldn't mind if it would be narrow in editing mode as well. you could also make it a toggle in case not everyone likes it? i always think that's a good idea for such specific style stuff, not everyone likes everything, but i'm not sure to which extent a toggle is possible.

**Comment by isaacfreeman** [comment]
>  i always think that's a good idea for such specific style stuff, not everyone likes everything

I agree, but also not everyone wants to spend their time customising every detail. I do, but I'm writing a whole theme anyway! I'd like Kakano to have a default appearance that has a good default appearance for people who don't care to tinker with the details, with settings for things that genuinely make a difference.

Based on this thread, it seems like line width in Reading View is one of those things! I've added a separate setting for it in version 4.1.0, so it can be different from Live Preview.

I'll add a credit to you in the README, with a link. I can use your Github profile, unless there's some other URL you'd prefer.

**Comment by incantatem1** [comment]
Sometimes editing themes can get a bit tedious, yeah... 

Thank you, the github profile is fine.
